data/json/items/tool/deployable.json

```diff
@@ -19,6 +19,25 @@
     "melee_damage": { "bash": 7 },
     "//2": "Based on https://www.amazon.com/dp/B0BQ6JNS2H"
   },
+  {
+    "id": "makeshift_brazier",
+    "type": "TOOL",
+    "name": { "str": "makeshift brazier" },
+    "description": "A protable brazier fashioned from a steel bucket by punching small holes on the sides to imrove air supply.  It is too small to fit anything much larger than splintered wood.  Fires set in the bucket will not spread to surrounding flammable objects",
```

Comment:
```suggestion
    "description": "A portable brazier fashioned from a steel bucket by punching small holes on the sides to improve the air supply.  It is too small to fit any fuel item larger than some splintered wood.  Fires set in this bucket will not spread to surrounding flammable objects",
```

Reply:
sorry, this is my first time using git and I just got so wrapped up in documentation that I completely forgot to proof read.